src/PostgREST/Router.hs

```diff
@@ -0,0 +1,42 @@
+module PostgREST.Router where
```

Comment:
as long as we have flat routes (/endpoint) and the support of single item routes (/endpoint/:id) i don't see the benefit of this code.
There is however something interesting in the "data Endpoint" data type. It might help in refactoring the App.hs code